server/app/api.py

```diff
@@ -967,59 +1244,43 @@ class GroupAPI(APIResource):
                 'members': KeyArg('User')
             }
         },
+        'invite': {
```

Comment:
What's the purpose of this endpoint? I can't seem to find the associated method.